packages/amplify-provider-awscloudformation/src/pre-push-cfn-processor/modifiers/s3-sse-modifier.ts

```diff
@@ -0,0 +1,21 @@
+import Bucket, { BucketEncryption, ServerSideEncryptionByDefault, ServerSideEncryptionRule } from 'cloudform-types/types/s3/bucket';
+import { ResourceModifier } from '../pre-push-cfn-modifier';
+
+export const applyS3SSEModification: ResourceModifier = async (resource: Bucket) => {
+  if (resource.Properties?.BucketEncryption) {
+    return; // don't overwrite existing encryption config if present
```

Comment:
Shouldn't this return the `resource`
```suggestion
    return resource; // don't overwrite existing encryption config if present
```